CASC/PortfolioMode.cpp

```diff
@@ -46,6 +47,14 @@ using namespace Lib;
 using namespace CASC;
 
 PortfolioMode::PortfolioMode() : _slowness(1.0), _syncSemaphore(2) {
+  unsigned cores = System::getNumberOfCores();
```

Comment:
Inline a function/member from `ScheduleExecutor` to track how many worker processes we should spin up.